### ISSUE TITLE
Doesn't work anymore and even if you debug it, OpenSea is using reCAPTCHA

### ISSUE THREAD

**Issue by jointheantfarm** [lakshyabatman/Bulk-uploader-NFT#11]

**Comment by lakshyabatman** [comment]
Yeah, sadly Opensea added captcha. I tried to bypass using extensions but it doesn't work out.
I'm open to discuss more about the same with other folks contributing or using this project.

**Comment by maximedrn** [comment]
Hello @lakshyabatman, being myself a developer of a project similar to yours, I have been confronted with reCAPTCHAs. Unfortunately using an extension like Buster doesn't help much, many NFT's won't be uploaded because Buster works with the audio functionality of reCAPTCHAs.

So I worked on my side. I can advise you to take a look at my [repository](https://github.com/maximedrn/opensea-automatic-bulk-upload-and-sale) which contains solutions to solve reCAPTCHAs.
